### PR TITLE
3540 frontend boilerplate

### PR DESCRIPTION
**Pull request by alibombaye**

### CODE REVIEW

File: Frontend/app/common/components/error-summary/macro.njk [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% from 'components/error-summary/macro.njk' import errorSummary as nhsErrorSummary %}
```
Comment (kat1906) [inline Frontend/app/common/components/error-summary/macro.njk]: are we going to use this?

Comment (kat1906) [reply Frontend/app/common/components/error-summary/macro.njk]: if yes we can move to BCL

File: Frontend/app/pages/error/template.njk [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% extends 'includes/layout.njk' %}
```
Comment (kat1906) [inline Frontend/app/pages/error/template.njk]: move to BCL

File: Frontend/app/styles/_app.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.bc-c-title {
+  border-bottom: 2px solid $color_nhsuk-blue;
```
Comment (kat1906) [inline Frontend/app/styles/_app.scss]: if these are duplicated then move to BCL

File: Frontend/app/test-utils/config.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+export const apiLocalhost = 'http://localhost:8080';
+
+export const apiPath = '/api/v1/Solutions/S100000-001';
```
Comment (kat1906) [inline Frontend/app/test-utils/config.js]: dont think this is correct

File: Frontend/package.json [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    "start": "cross-env NODE_ENV=production npm run build && babel-node server",
+    "test:ui": "cross-env NODE_ENV=test LOGGER_LEVEL=off babel-node testcafeRunner.js",
+    "test:integration": "cross-env NODE_ENV=pipeline LOGGER_LEVEL=off babel-node testcafeRunner.js",
+    "test": "jest --coverage",
+    "prewatch": "npm run build:css",
+    "watch": "concurrently \"npm run watch:scss\" \"npm run watch:js\"  \"npm start:dev\"",
+    "watch:scss": "npm run build:css -- -w",
+    "watch:js": "webpack --w",
+    "webpack": "webpack",
+    "debug": "babel-node server --inspect"
```
Comment (kat1906) [inline Frontend/package.json]: we could get rid of all the commands we don't use

File: Frontend/app/common/components/error-summary/macro.njk (hunk 1 above)
Comment (alibombaye) [reply Frontend/app/common/components/error-summary/macro.njk]: yes we are 😒.  OK to do it after this PR?

File: Frontend/app/pages/error/template.njk (hunk 2 above)
Comment (alibombaye) [reply Frontend/app/pages/error/template.njk]: Move the template?

Comment (kat1906) [reply Frontend/app/pages/error/template.njk]: component

File: Frontend/app/styles/_app.scss (hunk 3 above)
Comment (alibombaye) [reply Frontend/app/styles/_app.scss]: I need to remove these. They are not being used.

**Pull request merged**